adafruit_dash_display.py

```diff
@@ -22,13 +22,19 @@
   https://github.com/adafruit/circuitpython/releases
 
 """
+try:
+    from typing import Tuple, Callable, Optional, Any
+except ImportError:
+    pass
 
 import time
 from collections import OrderedDict
-from adafruit_display_shapes.rect import Rect
-from adafruit_display_text.label import Label
 import displayio
 import terminalio
+import digitalio
+from adafruit_display_shapes.rect import Rect
+from adafruit_display_text.label import Label
+from adafruit_io.adafruit_io import IO_MQTT
```

Comment:
since `IO_MQTT` is used only for typing we can put it up inside of the try / except block with the other typing imports:

```
try:
    from typing import Tuple, Callable, Optional, Any
    from adafruit_io.adafruit_io import IO_MQTT
except ImportError:
    pass
```
That way when this runs on the microcontroller it will save RAM by not importing that since it's not used at runtime for the actual execution of the code.